docs/articles/documentation/using-testcafe/programming-interface/TestCafe.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+title: TestCafe Class
+permalink: /documentation/using-testcafe/programming-interface/TestCafe/
+---
+# TestCafe Class
+
+A TestCafe server instance.
+
+To create it, use the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/global/#createTestCafe) global function.
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+```
```

Comment:
This does not describe `createTestCafe` - it describes the TestCafe class.
Args are listed where `createTestCafe` is described - `/testcafe/documentation/using-testcafe/programming-interface/global/#createTestCafe`